cockatrice/src/user_context_menu.cpp

```diff
@@ -89,6 +94,27 @@ void UserContextMenu::banUser_processUserInfoResponse(const Response &r)
     dlg->show();
 }
 
+void UserContextMenu::adjustMod_processUserResponse(const Response &resp, const CommandContainer &commandContainer)
```

Comment:
I'm not sure the user menu is the right place to handle user level adjustment responses